core/station/api/spec.did

```diff
@@ -1567,6 +1603,23 @@ type ChangeCanisterResourceAction = variant {
   Create;
 };
 
+// The target of a request to create a managed canister.
+type CreateManagedCanisterResourceTarget = variant {
+  Any;
+};
+
+// The target canister of a request to change a managed canister.
+type ChangeManagedCanisterResourceTarget = variant {
+  Any;
+  Canister : principal;
+};
+
+// The actions that are available for managed canisters.
+type ManagedCanisterResourceAction = variant {
+  Create : CreateManagedCanisterResourceTarget;
```

Comment:
is the plan here to extend in the future so that users would be able to set an existing canister? to, for example be able to import an existing one?

Reply:
The future extension I thought about are specific subnets.